monkey/monkey_island/cc/services/reporting/zero_trust_service.py

```diff
@@ -1,21 +1,26 @@
-import json
+from typing import List
 
 import common.data.zero_trust_consts as zero_trust_consts
+from bson.objectid import ObjectId
 
 from monkey_island.cc.models.zero_trust.finding import Finding
 
+# How many events of a single finding to return to UI.
+# 50 will return 50 latest and 50 oldest events from a finding
+EVENT_FETCH_CNT = 50
```

Comment:
10? I'm afraid user won't get enough variety in 20 events. I'd like to keep it to the larger end of the spectrum, because reading 100 events on UI is still more usable than having to export 100 events and then parse the json somehow.